packages/app/src/explorer/utils.ts

```diff
@@ -26,3 +28,52 @@ export function needsNxBadge(
 
   return false;
 }
+
+function getExplorerButtonList(): HTMLButtonElement[] {
+  const explorer = document.querySelector(`#${EXPLORER_ID}`);
+  if (!explorer) {
+    return [];
+  }
+  return [...explorer.querySelectorAll('button')];
+}
+
+export function focusParent(e: { preventDefault: () => void }, path: string) {
+  const buttonList = getExplorerButtonList();
+  const parentPath = path.slice(0, path.lastIndexOf('/')) || '/';
+
+  const parentButton = buttonList.find(
+    (element) => element.dataset.path === parentPath
```

Comment:
https://github.com/sindresorhus/eslint-plugin-unicorn/blob/v43.0.2/docs/rules/prefer-dom-node-dataset.md

Reply:
Haha forgot about this API, thanks! Though given that we deal with other kinds of datasets, a small comment would maybe avoid confusion:

```suggestion
    (element) => element.dataset.path === parentPath // `data-path` attribute
```

Reply:
In this case, we can disable the rule then and use `getAttribute` instead